Remora.Rest/Http/RestHttpClient.cs

```diff
@@ -651,7 +651,7 @@ void IRestCustomizable.RemoveCustomization(RestRequestCustomization customizatio
         }
 
         // See if we have a JSON error to get some more details from
-        if (response.Content.Headers.ContentLength is not > 0)
+        if (response.Content.Headers.ContentLength == 0)
```

Comment:
Somewhat of a nitpick, but shouldn't this be `is 0` for style/consistency

Reply:
Up to you, I was matching line 589 further up in the file that uses the same pattern. Whatever is decided on though I think those 2 checks should probably match.

Reply:
Yeah, stick to the same as the other check - we can do a pass over constants + pattern matching later.